central/notifiers/cscc/cscc.go

```diff
@@ -169,7 +191,7 @@ func (c *cscc) AlertNotify(ctx context.Context, alert *storage.Alert) error {
 
 	cluster, err := c.getCluster(alert.GetDeployment().GetClusterId())
 	if err != nil {
-		return err
+		return findingID, nil, err
```

Comment:
[nit] In this case I would still return empty string because if err != nil, then it's not meant to be used. Doesn't make a functional difference though.